libs/api/domains/signature-collection/src/lib/dto/area.input.ts

```diff
@@ -0,0 +1,9 @@
+import { IsString } from 'class-validator'
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class AreaInput {
```

Comment:
Mutations, Queries, Types and Models needs to be prefixed with the api domain name.

https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api

You can either name the class `SignatureCollectionAreaInput` or use the `@InputType` decorator to control the GQL schema name and keep the class name short as it is.

```
@InputType('SignatureCollectionAreaInput')
export class AreaInput {
```